src/Highway.Data.EntityFramework/Interfaces/IEntityDataContext.cs

```diff
@@ -1,22 +1,11 @@
-﻿using System.Collections.Generic;
-using System.Data.Common;
+﻿using System.Data.Common;
 
 namespace Highway.Data.EntityFramework
 {
     /// <summary>
     /// </summary>
-    public interface IEntityDataContext : IDataContext
```

Comment:
It could be argued that moving this method isn't safe, if it is just a SQL string and params.  But the _intent_ of the name "ExecuteSqlQuery" seems obvious.  Thoughts?